frontend/recorder/save_screen.js

```diff
@@ -144,6 +144,9 @@ class SaveScreen extends React.PureComponent {
         <FormGroup label="Target">
           <HTMLSelect options={grantOptions} value={targetUrl} onChange={this.handleTargetChange} />
         </FormGroup>
+        <FormGroup labelFor='recordNameInput' label={"Recording Name"}>
+          <input id='recordNameInput' type='text' maxlength='30' placeholder="recording name here" className='bp3-input' onChange={this.handleNameChange} required/>
```

Comment:
Here it should be `maxLength` instead of `maxlength`. As it is showing some errors in the console.

Reply:
done